### ISSUE TITLE
Feature Request: Custom Fonts

### ISSUE THREAD

**Issue by ClockGen** [gsantner/markor#464]
Would be nice to be able to load custom .ttf.

**Comment by ali-hardan** [comment]
In preview mode, I have done it using "custom CSS".

**Comment by def14nt** [comment]
@gsantner I think he means a UI should be implemented for this, not CSS. That's a good idea though, maybe allowing users to import stylesheets for the preview would be a good idea? Any thoughts are appreciated.

**Comment by ClockGen** [comment]
Yes, I meant UI, considering you already can select predefined fonts for editor, but custom ones would be great. Since markor primarily a text editor, and user spends majority of time interacting with text, IMO it's important to let users set their own typeface that's comfortable for them. For example, I don't use preview mode at all, only when I want to share the document or create a shortcut. I wish I could implement this myself, but I have little experience with development for android.

**Comment by gsantner** [comment]
Community: Feel free to work on it ;)

**Comment by gsantner** [comment]
Put fonts to `markor/.app/fonts/*.ttf` and wait for the next update

**Comment by Secret-chest** [comment]
Feature this better in the settings.
